### PR TITLE
Laravel 5.2

### PR DESCRIPTION
**Pull request by nanaya**
And a bunch of other updates.

### CODE REVIEW

File: app/Http/Controllers/Controller.php [hunk 1]
```diff
@@ -19,13 +19,15 @@
  */
 namespace App\Http\Controllers;
 
-use Illuminate\Foundation\Bus\DispatchesCommands;
+// use Illuminate\Foundation\Auth\Access\AuthorizesRequests;
+// use Illuminate\Foundation\Auth\Access\AuthorizesResources;
```
Comment (nekodex) [inline app/Http/Controllers/Controller.php]: 😕

File: composer.json [hunk 2]
```diff
@@ -2,36 +2,43 @@
   "name": "ppy/osu-web",
   "description": "osu!web",
   "type": "project",
-  "homepage": "https://jizz.ppy.sh",
+  "homepage": "https://new.ppy.sh",
   "license": "AGPL-3.0+",
   "support": {
     "email": "jizz@ppy.sh",
```
Comment (nekodex) [inline composer.json]: 👍

Comment (nanaya) [inline composer.json]: shall I replace this with `noreply@ppy.sh`?

Comment (peppy) [inline composer.json]: support@ppy.sh is fine

**Pull request merged**